studio/deskStructure.js

```diff
@@ -27,10 +23,32 @@ export default () =>
         )
         .icon(MdHome),
       S.listItem()
-        .title('Pages')
-        .schemaType('page')
-        .child(S.documentTypeList('page').title('Page'))
-        .icon(MdLayers),
+        .title('Services Page')
+        .schemaType('service')
+        .child(
+          S.editor()
+          .id('service')
+          .schemaType('service')
+          .documentId('service'))
+        .icon(MdAssignment),
+      S.listItem()
+        .title('About Page')
+        .schemaType('about')
+        .child(
+          S.editor()
+          .id('about')
+          .schemaType('about')
+          .documentId('about'))
+        .icon(MdErrorOutline),
+      S.listItem()
+        .title('Contacts Page')
+        .schemaType('contact')
+        .child(
+          S.editor()
+          .id('contact')
+          .schemaType('contact')
+          .documentId('contact'))
+        .icon(MdContactMail),
```

Comment:
```suggestion
        .title('Services Page')
        .schemaType('service')
        .child(
          S.editor()
          .id('service')
          .schemaType('page')
          .documentId('service'))
        .icon(MdAssignment),
      S.listItem()
        .title('About Page')
        .schemaType('about')
        .child(
          S.editor()
          .id('about')
          .schemaType('page')
          .documentId('about'))
        .icon(MdErrorOutline),
      S.listItem()
        .title('Contacts Page')
        .schemaType('contact')
        .child(
          S.editor()
          .id('contact')
          .schemaType('page')
          .documentId('contact'))
        .icon(MdContactMail),
```

«Don't model your content for presentation»: Read more about it here: sanity.io/docs/donts-and-dos